modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/DividedProgress.scala

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.web.client.components
+
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common.implicits._
+import react.common.Css
+import cats.implicits._
+import seqexec.web.client.semanticui.elements.progress.Progress
+import web.client.ReactProps
+
+/**
+  * Progress bar divided in steps
+  */
+final case class DividedProgress(
+                                  sections            : List[DividedProgress.Label],
+                                  sectionTotal        : DividedProgress.Quantity,
+                                  value               : DividedProgress.Quantity,
+                                  indicating          : Boolean = false,
+                                  progress            : Boolean = false,
+                                  completeSectionColor: Option[String] = None,
+                                  ongoingSectionColor : Option[String] = None,
+                                  progressCls         : List[Css] = Nil,
+                                  barCls              : List[Css],
+                                  labelCls            : List[Css] = Nil
+                                ) extends ReactProps {
+  @inline def render: VdomElement = DividedProgress.component(this)
+}
+
+object DividedProgress {
+  type Props = DividedProgress
+
+  type Label = String
+  type Quantity = Int
+
+  implicit val propsReuse: Reusability[Props] = Reusability.byRef

```

Comment:
I think this should be done with `Reusability.derive[DivideProgress]`, otherwise `byRef` will be essentially always `false`

Reply:
I was getting warnings in the console about that and had it pending to look into it. Fixed now, thank you!